Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -492,7 +493,35 @@ await useCase.Execute((organisationId, invitePersonToOrganisation))
                 return operation;
             });
 
-        app.MapDelete("/{organisationId}/person-invite/{personInviteId}",
```

Comment:
I don't understand why you are removing the delete endpoint here. This was intentional piece of work that I did before I went on holiday.

Reply:
Delete endpoint is not removed, it is there in the API